src/test/java/LoginTests.java

```diff
@@ -1,115 +1,70 @@
-        submitLogin.click();
-        WebElement avatar = driver.findElement(By.cssSelector(".avatar"));
-        Assert.assertTrue(avatar.isDisplayed());
-        driver.quit();
-    }
+    public static void LoginEmptyEmailPasswordTest() {
+        LoginPage loginPage = new LoginPage(getThreadLocal());
 
+        loginPage.provideEmail("")
+                .providePassword("te$t$tudent")
+                .clickSubmitBtn();
 
-    @Test
-    public static void LoginWrongPasswordTest () {
-
-        WebDriver driver = new ChromeDriver();
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
-
-        String url = "https://bbb.testpro.io/";
-        driver.get(url);
-        WebElement emailInput = driver.findElement(By.cssSelector("[type='email']"));
-        emailInput.click();
-        emailInput.sendKeys("demo@class.com");
-        WebElement passwordInput = driver.findElement(By.cssSelector("[type='password']"));
-        passwordInput.click();
-        passwordInput.sendKeys("student");
-        WebElement submitLogin = driver.findElement(By.cssSelector("button[type='submit']"));
-        submitLogin.click();
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-        Assert.assertTrue(submitLogin.isDisplayed());
-        driver.quit();
+        Assert.assertTrue(loginPage.getRegistrationLink().isDisplayed());
     }
 
+    @Test
+    public static void LoginWrongPasswordTest() {
+        LoginPage loginPage = new LoginPage(getThreadLocal());
+
+        loginPage.provideEmail("ksenia.strigkova@gmail.com")
+                .providePassword("te$t123")
+                .clickSubmitBtn();
+
+        Assert.assertTrue(loginPage.getRegistrationLink().isDisplayed());
+    }
 
     @Test
-    public static void LoginEmptyPasswordTest () {
-
-        WebDriver driver = new ChromeDriver();
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
-
-        String url = "https://bbb.testpro.io/";
-        driver.get(url);
-        WebElement emailInput = driver.findElement(By.cssSelector("[type='email']"));
-        emailInput.click();
-        emailInput.sendKeys("demo@class.com");
-        WebElement submitLogin = driver.findElement(By.cssSelector("button[type='submit']"));
-        submitLogin.click();
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-        Assert.assertTrue(submitLogin.isDisplayed());
-        WebElement registationLink = driver.findElement(By.id("hel"));
-        Assert.assertTrue(registationLink.isDisplayed(), "==== Registation link displayed ====");
-        driver.quit();
+    public static void LoginEmptyPasswordTest() {
+        LoginPage loginPage = new LoginPage(getThreadLocal());
+
+        loginPage.provideEmail("ksenia.strigkova@gmail.com")
+                .providePassword("")
+                .clickSubmitBtn();
+
+        Assert.assertTrue(loginPage.getRegistrationLink().isDisplayed());
     }
 
     @Test
-    public static void LoginWrongEmailTest () {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

```
    public void LoginSucceedTest()
    public static void LoginEmptyEmailPasswordTest()
    LoginWrongPasswordTest()
    LoginEmptyPasswordTest()
    LoginWrongEmailTest()
    logOut()
```

For example **loginWrongEmailTest**

![image](https://user-images.githubusercontent.com/104761482/222778290-8a79e9cb-f540-4daa-aff0-eda8bcc847b1.png)